guide/additional-info/hosting.md

```diff
@@ -0,0 +1,68 @@
+# Hosting your bot on a VPS
+
+A bot requires itself to be hosted in order for it to run, however, it would be unideal to host it on your computer all day without shutting it down. That's where a **Virtual Private Server**(VPS) comes into place. A good VPS can mark a turning point for your project as you can host local databases, and it is reliable as well. A VPS suitable for your Discord bot is typically cheap, ranging from around $2-10/month.
```

Comment:
```suggestion
A bot requires itself to be hosted in order for it to run, however, it would not be ideal to host it on your computer all day without shutting it down. That's where a **Virtual Private Server** (VPS) comes into place. A good VPS can mark a turning point for your project as you can host local databases, and it is reliable as well. A VPS suitable for your Discord bot is typically cheap, ranging from around $2-10/month.
```